va/va.h

```diff
@@ -1147,12 +1147,16 @@ typedef struct _VAConfigAttrib {
 
 /** @name Attribute values for VAConfigAttribDecJPEG */
 /**@{*/
+/** \brief Driver supports region-of-interest (ROI) decode */
+#define VA_DEC_ROI                     0x00000004
```

Comment:
seems , no need to define this macro, just need add description with "uint32_t roi:1"
like /* 1. is support 0.unsupport

Reply:
okay, got it , thanks for the comment I will update this.

Reply:
Pushed the updated version.